src/test/java/domain/PurchasedLottosTest.java

```diff
@@ -0,0 +1,55 @@
+package domain;
+
+import domain.strategy.CustomPurchaseStrategy;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@DisplayName("상금과 수익률을 옳바르게 계산하는지 테스트한다.")
+public class PurchasedLottosTest {
+
+    PurchasedLotto lottos;
+    WinningNumber winningLotto;
+    int inputMoney;
+
+    @BeforeEach
+    void init() {
+        inputMoney = 1400;
+        lottos = new PurchasedLotto(inputMoney);
+        lottos.purchase(new CustomPurchaseStrategy(Arrays.asList(new Integer[]{1, 2, 3, 43, 44, 45})));
```

Comment:
```suggestion
        lottos.purchase(new CustomPurchaseStrategy(Arrays.asList(1, 2, 3, 43, 44, 45)));
```

IntelliJ를 사용하고 있다면 조금 더 주의를 기울여 볼까요?

Reply:
해당 코드도 `Redundant array creation for calling varargs method` warning메시지를 띄워줬네요 😅

앞으로 intelliJ가 주는 메시지들에 더욱 신경을 써보겠습니다🔥